index.html

```diff
@@ -209,5 +209,18 @@ <h2 id="heading-2">Skills</h2>
       <hr>
     </section>
 </main>
+<section class="form">
+  <div class="about-me">
+    <h1 class="heading1"  >Contact me</h1>
+    <P class="para-1" >If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it</P>
+  </div>
+  <form action="https://formspree.io/f/xeqwglky" method="POST">
+    <input type="text" value="Diana"  maxlength="30"  placeholder="Name" class="input text" required/>
+    <input type="email" value="dianab21@gmail.com" placeholder="Email" class="input text" required/>
```

Comment:
Great job implementing the form section of the project 👍. kindly make sure you have the name attribute for your inputs because only form elements with a name attribute will have their values passed when submitting a form. 🧔 😊 .

Also, kindly style your input in line with the Figman template design

| Your Design | Figma Design |
| :---:   | :---: | 
|<img width="381" alt="image" src="https://user-images.githubusercontent.com/73854999/220569339-5d10290e-19b3-48c0-ac7f-9d9700ecd7b7.png">|<img width="381" alt="image" src="https://user-images.githubusercontent.com/73854999/220569211-d21bc7ba-daad-4a40-8a64-3c44e022f530.png"> |